benchmarks/precision_ab_vcache_vs_ubf16.py

```
@@ -0,0 +1,79 @@
+          accumulated fp32, ONE bf16 rounding at the state store (our kernel's
+          rounding points) — plus the ACTUAL vcache kernel for cross-check.
+  U-BF16: the SAME fp32-solved u ROUNDED TO BF16 (u-ring dtype), then the same
+          fp32 fold + one bf16 store round (#4081's payload rounding point).
+The only difference between RAW-V and U-BF16 is the u bf16 rounding.
+
+Run (from the repo root): python benchmarks/precision_ab_vcache_vs_ubf16.py"""
+import importlib.util, math, os, sys, torch
+REPO=os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, os.path.join(REPO,"tests","gdn"))
+def load(n,p):
+    s=importlib.util.spec_from_file_location(n,p);m=importlib.util.module_from_spec(s);s.loader.exec_module(m);return m
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)